libpyvinyl/DiffractionData/api_spec_v2.py

```diff
@@ -0,0 +1,74 @@
+"""
+The following documents how the various classes in libpyvinyl and a IO class to be implemented
+are supposed to be used.
+"""
+
+import InteractionCalculator, DiffractionCalculator, DataCollection
+import WavefrontData, SampleData
+
+
+
+base_path = 'my_base'
+interaction = InteractionCalculator()
+interaction.base_path = base_path
+interaction.input = DataCollection(WavefrontData.file('prop.h5',format='wpg'), SampleData.file('sample.pdb',format='pdb'))
+interaction.output_suffix = 'interaction'
+interaction.backengine()
+
+diffraction = DiffractionCalculator()
+diffraction.base_path = base_path
+diffraction.input = interaction.output
+diffraction.output_suffix = 'diffr'
```

Comment:
Can set `base_path`, `input`, `output_suffix` through the constructor (`__init__()`)? Otherwise I would find it overly complex. Using `__init__()` we could also configure sensible defaults for each derived calculator.